.gitignore

```diff
@@ -1,3 +1,7 @@
+# Saved models
+/saved*
```

Comment:
If you add `/saved*`, the `saved/` will be duplicated. Maybe you need to delete the latter.

Reply:
Oh I fotgot to squash my commits